pkg/operator/ceph/cluster/cluster.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/rook/rook/pkg/operator/ceph/cluster/mgr"
 	"github.com/rook/rook/pkg/operator/ceph/cluster/mon"
 	"github.com/rook/rook/pkg/operator/ceph/cluster/osd"
+	"github.com/rook/rook/pkg/operator/ceph/cluster/rbd"
```

Comment:
I had my own debate about this as well. My thought was that if there are more features related to rbd in the future we would include them in the same rbd package instead of creating a package for each specific feature. This also follows the same naming pattern as other packages.